prep-html-syntax/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<body>
```

Comment:
Hi Joel, be sure to include a `<head></head>` element before your `<body></body>` element.

Reply:
Ok will do! For some reason, when I was merging the skeleton exercise this got merged as well so I wasn't able to update it. I understand the concept completely though.